tools/cbmc/proofs/HTTP/IotHttpsClient_ReadHeader/IotHttpsClient_ReadHeader_harness.c

```diff
@@ -18,6 +18,9 @@ void harness() {
   initialize_IotResponseHandle(respHandle);
   if (respHandle) {
     __CPROVER_assume(is_valid_IotResponseHandle(respHandle));
+    // ???: replace assumption above with initialization
+    respHandle->httpParserInfo.readHeaderParser.data = respHandle;
```

Comment:
Would this be better in `allocate_IotResponseHandle()`?

Reply:
Same as above.